deploy/olm-catalog/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -697,7 +697,8 @@ metadata:
       cjoge30iLmZvcm1hdChlcnIpKQo=
     external.features.ocs.openshift.io/validation: '{"secrets":["rook-ceph-operator-creds",
       "rook-csi-rbd-node", "rook-csi-rbd-provisioner"], "configMaps": ["rook-ceph-mon-endpoints",
-      "rook-ceph-mon"], "storageClasses": ["ceph-rbd"]}'
+      "rook-ceph-mon"], "storageClasses": ["ceph-rbd"], "CephCluster": ["MonitoringEndpoint", 
+      "MonitoringPort"]}'
```

Comment:
Where is this `"MonitoringPort"` coming from? I can't see it in CSV merger.

Reply:
Missed it. I will update